pkg/controller/clusterdeployment/installconfigvalidation.go

```diff
@@ -2,9 +2,10 @@ package clusterdeployment
 
 import (
 	"github.com/pkg/errors"
-	"gopkg.in/yaml.v2"
+	"sigs.k8s.io/yaml"
```

Comment:
NTR: yaml.v2 seems not to work properly for unmarshalling an install-config. Some (but not all) fields are unrecognized and end up unset. That makes me leery of using it at all anywhere, so a subsequent PR will purge other uses of it in this project. This PR changes just this one instance to get the feature working.